frontend/src/components/OdhExploreCard.tsx

```diff
@@ -55,37 +47,41 @@ const OdhExploreCard: React.FC<OdhExploreCardProps> = ({
       data-id={odhApp.metadata.name}
       id={odhApp.metadata.name}
       role="listitem"
-      isHoverable={!disabled}
       isSelectable={!disabled}
       isSelected={isSelected}
       className={cardClasses}
       onClick={() => !disabled && onSelect()}
     >
-      <CardHeader>
-        <CardHeaderMain style={{ maxWidth: '33%', width: '100%' }}>
-          <BrandImage src={odhApp.spec.img} alt={odhApp.spec.displayName} />
-        </CardHeaderMain>
-        {!dashboardConfig.spec.dashboardConfig.disableISVBadges && (
-          <CardActions hasNoOffset>
-            <Flex
-              spaceItems={{ default: 'spaceItemsSm' }}
-              alignItems={{ default: 'alignItemsCenter' }}
-              direction={{ default: 'column' }}
-            >
-              {odhApp.spec.comingSoon && (
-                <FlexItem className="odh-card__coming-soon">Coming soon</FlexItem>
-              )}
-              {!odhApp.spec.comingSoon && odhApp.spec.category && (
-                <FlexItem className={badgeClasses}>
-                  <OdhExploreCardTypeBadge category={odhApp.spec.category} />
-                </FlexItem>
-              )}
-              {odhApp.spec.beta && (
-                <FlexItem className="odh-card__partner-badge odh-m-beta">Beta</FlexItem>
-              )}
-            </Flex>
-          </CardActions>
-        )}
+      <CardHeader
+        {...(!dashboardConfig.spec.dashboardConfig.disableISVBadges && {
+          actions: {
+            actions: (
+              <>
```

Comment:
Fragments are not needed when dealing with 1 child... they help when you have two or more siblings and you don't have a comment parent (usually only needed when passing as a "single prop" or "returning as a single value").